packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -56,16 +56,19 @@ class CpuProfileData {
         jsonDecode(jsonEncode(json[stackFramesKey] ?? <String, dynamic>{}));
     for (final MapEntry<String, dynamic> entry in stackFramesJson.entries) {
       final stackFrameJson = entry.value;
-      final stackFrame = CpuStackFrame(
+      final resolvedUrl = stackFrameJson[resolvedUrlKey] ?? '';
+      final processedUrl = stackFrameJson[processedUrlKey] ?? resolvedUrl;
```

Comment:
rename to `resolvedPackageUri`. Same comment for the key const: `processedUrlKey` --> `resolvedPackageUriKey`